.github/workflows/test_main_macos.yml

```diff
@@ -0,0 +1,58 @@
+name: test_macos
```

Comment:
I guess, this file is irrelevant because it runs only when merging into `main`, and this branch won't actually be merged into `main` anytime soon.